flake8_type_checking/checker.py

```diff
@@ -332,7 +332,7 @@ def import_name(self) -> str:
     @property
     def import_type(self) -> ImportTypeValue:
         """Return the import type of the import."""
-        return cast(ImportTypeValue, classify_import(self.full_name))
+        return cast(ImportTypeValue, classify_base(self.full_name.partition('.')[0]))
```

Comment:
This will behave exactly the same using `split`, right? Think `split` might be easier to understand for most (me 😅) 
```suggestion
        return cast(ImportTypeValue, classify_base(self.full_name.split('.')[0]))
```

Reply:
Ah, I used it as it is/was what is used internally in `classify-imports`, likely as a micro-optimisation:

```console
❯ python -m timeit -s 'x="a.b.c.d.e"' 'x.partition(".")[0]'
5000000 loops, best of 5: 50.1 nsec per loop
❯ python -m timeit -s 'x="a.b.c.d.e"' 'x.split(".")[0]'
5000000 loops, best of 5: 61.1 nsec per loop
❯ python -m timeit -s 'x="a.b.c.d.e"' 'x.split(".", 1)[0]'
5000000 loops, best of 5: 57.9 nsec per loop
```

Did you still want me to change it? If so, we will want to limit to one split.

Reply:
Some confounding added data here for our consideration :sweat_smile: 

```console
❯ python -m timeit -s 'x="a.b.c.d.e"' 'x.partition(".")[0]'
5000000 loops, best of 5: 82.3 nsec per loop

~ took 2s
❯ python -m timeit -s 'x="a.b.c.d.e"' 'x.split(".")[0]'
5000000 loops, best of 5: 84.2 nsec per loop

~ took 2s
❯ python -m timeit -s 'x="a.b.c.d.e"' 'x.split(".", 1)[0]'
5000000 loops, best of 5: 97.6 nsec per loop
```
Think the simple `split` would be the easiest to parse for maintainers of the code, so I'd prefer that, but I don't feel very strongly about it :+1: Feel free to keep it as is if you want, and I'll merge this tomorrow when I get a chance :slightly_smiling_face: